pkg/crc/machine/machine.go

```diff
@@ -255,17 +255,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	}
 
 	// Check the certs validity inside the vm
+	needsCertsRenewal := false
 	logging.Info("Verifying validity of the cluster certificates ...")
 	expiringIn7Days, duration, err := cluster.CheckCertsValidity(sshRunner)
 	if err != nil {
-		result.Error = err.Error()
-		return *result, errors.New(err.Error())
-	}
-	// Only show when VM is started from stopped state.
-	if exists {
-		if expiringIn7Days {
+		//result.Error = err.Error()
```

Comment:
@cfergeau I think these comments shouldn't be there right?

Reply:
yeah left overs :)